webapp/src/components/NotificationStructure/NotificationStructure.js

```diff
@@ -38,6 +37,11 @@ const NotificationStructure = ({ id, title, description, state, dateAndTime }) =
     response()
   }, [description])
 
+  useEffect(() => {
+    const hour = parseInt(dateAndTime.substring(11, 13)) - 6
+    setTime(hour.toString() + dateAndTime.substring(13, 19))
```

Comment:
@steph222  what if we handle dateAndTime as Date type, because the string format may change depending on the date format so if the manipulate the date as Date type when solved that possible problem.

Nota: Date type also provide the getDate, getMonth and other to get needed information. Take a look at [this](https://www.w3schools.com/js/js_date_methods.asp)

Reply:
It was done that way because what is saved in the notification is a string, but of everything it contains, only two characters are needed, which are the hours, the rest is used normal.

Apply the change only in the notification

Reply:
@steph222 I suppose to, but in line 41, you are taking `dateAndTime.substring(11, 13)` that will really depend on the date format, despite of you need only one data from date I thing you should use that data in date format